tests/core/Jalasoft.TeamUp.Resumes.Core.Tests/UpdateResumeCoreTests.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace Jalasoft.TeamUp.Resumes.Core.Tests
+{
+    using System;
+    using System.Collections.Generic;
+    using Jalasoft.TeamUp.Resumes.Core;
+    using Jalasoft.TeamUp.Resumes.Core.Interfaces;
+    using Jalasoft.TeamUp.Resumes.DAL.Interfaces;
+    using Jalasoft.TeamUp.Resumes.Models;
+    using Moq;
+    using Xunit;
+
+    public class UpdateResumeCoreTests
+    {
+        private readonly ResumesService resumesService;
+
+        private readonly Mock<IResumesInMemoryRepository> mockResumeRepository;
+
+        public UpdateResumeCoreTests()
+        {
+            this.mockResumeRepository = new Mock<IResumesInMemoryRepository>();
+            this.resumesService = new ResumesService(this.mockResumeRepository.Object);
+        }
+
+        public static Resume GetResume()
+        {
+            var stubSkill = new Skill[]
+            {
+                new Skill
+                {
+                    Id = 1,
+                    Name = "c#"
+                },
+                new Skill
+                {
+                    Id = 1,
+                    Name = "devopsTest"
+                }
+            };
+
+            var resume = new Resume()
+            {
+                Id = 1,
+                Skills = stubSkill
+            };
+
+            return resume;
+        }
+
+        [Fact]
+        public void UpdateResume_Returns_Resume()
```

Comment:
You need to change the name for this test following this section https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices
[name of the method being tested]_[The scenario]_[The expected behavior when the scenario is invoked]
like this UpdateResume_validResume_Resume